src/lib/openapi/spec/create-feature-schema.ts

```diff
@@ -13,7 +13,13 @@ export const createFeatureSchema = {
             description: 'Unique feature name',
         },
         type: {
-            type: 'string',
+            enum: [
```

Comment:
Nice! Do we have this defined somewhere else that we could use instead of redefining it? Not a big deal, of course, but might save us from drift in the future 🤷🏼